### ISSUE TITLE
you do not need to run as root to have a lower port by using capabilities

### ISSUE THREAD

**Issue by f1-outsourcing** [matrix-org/synapse#7962]
you do not need to run as root to have a lower port by using capabilities

https://github.com/matrix-org/synapse#id12

https://man7.org/linux/man-pages/man7/capabilities.7.html

**Comment by anoadragon453** [comment]
You're right in that it's a bit of a weak justification. A more useful one may be that it's required for [worker setups](https://github.com/matrix-org/synapse/blob/master/docs/workers.md) and/or allows you to run multiple matrix-related services behind the same hostname/port, such as a [separate media repo implementation](https://github.com/turt2live/matrix-media-repo/).

**Comment by reivilibre** [comment]
authbind can also be used for these cases, but the trouble is that we are already overloading the admin with new concepts and it is hard to support all of them. If admins who already know about these tools want to use them, that's fine and usually doesn't result in us needing to provide support for that.

I think we can leave it as is, as a little innocent lie(?).
